.gitignore

```diff
@@ -15,3 +15,9 @@ resources/IndexData/index/*.cfe
 resources/IndexData/index/*.cfs
 resources/IndexData/index/*.si
 resources/IndexData/index/segments*
+
+.vs/*
```

Comment:
what for?

Reply:
some files are generated by Visual Studio IDE

Reply:
ok jusjt keep it, but please remove the `.vscode` part.